public/app/plugins/panel/singlestat/module.ts

```diff
@@ -590,11 +590,7 @@ class SingleStatCtrl extends MetricsPanelCtrl {
         const color = getColorForValue(data, data.display.numeric);
         if (color) {
           getPanelContainer().css('background-color', color);
-          if (scope.fullscreen) {
-            elem.css('background-color', color);
-          } else {
-            elem.css('background-color', '');
-          }
+          elem.css('background-color', '');
```

Comment:
There's an if statement missing here, intentional?

Reply:
It was, couldn't get it to be set but I might revert this one since this panel is deprecated. Good feedback!